<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings Diff</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="gbFilter">
          <property name="title">
           <string>Filter</string>
          </property>
          <layout class="QVBoxLayout" name="filterLayout">
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidget"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Info</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="lblStatus">
             <property name="text">
              <string>No files selected.</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,1">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Reason: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblReason">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Type 1: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblType1">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Shape 1: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblShape1">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Type 2: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblType2">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
             <property name="topMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <weight>75</weight>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>Shape 2: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="lblShape2">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblFile1">
          <property name="text">
           <string>[FILE1]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="tbValue1">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblFile2">
          <property name="text">
           <string>[FILE2]</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPlainTextEdit" name="tbValue2"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
